packages/scene-composer/src/utils/entityModelUtils/sceneUtils.ts

```diff
@@ -97,6 +99,46 @@ export const isDynamicScene = (document?: ISceneDocumentInternal): boolean => {
   return !isEmpty(document?.properties?.[KnownSceneProperty.SceneRootEntityId]);
 };
 
+export const useSceneRootEntityId = (document?: ISceneDocumentInternal): string => {
```

Comment:
May have linter issues as use<functionName> implies it's a react hook but this isn't.